base.go

```diff
@@ -0,0 +1,765 @@
+package bacnet
```

Comment:
In general, I am not a big fan of putting all these consts in one place, especially if they are not used all over. Maybe it would better to define them only in the package where you actually use them.

Maybe they are needed all over, IDKm but if something is defined here, and then used in just one file or package, then it does not make sense...